### PR TITLE
Feature/piloting data

### PR DESCRIPTION
**Pull request by bbonf**
Addresses #44

### CODE REVIEW

File: experiments/models.py [hunk 1]
```diff
@@ -183,6 +185,16 @@ def __str__(self):
     session = models.ForeignKey(
         'ParticipantSession', on_delete=models.CASCADE, null=True)
 
+    @property
+    def status_label(self):
+        test = _('experiments:models:datapoint:label:test')
+        pilot = _('experiments:models:datapoint:label:pilot')
```
Comment (tymees) [inline experiments/models.py:191]: I'd rather have these values in constants somewhere (Top of the file, top of the class, both could work)

File: experiments/utils/download.py [hunk 2]
```diff
@@ -134,7 +142,11 @@ def _create_zip(
     with zipfile.ZipFile(buffer, "a", zipfile.ZIP_DEFLATED, True) as zip_file:
         export_report = EXPORT_REPORT_HEADER
 
-        for dataPoint in experiment.datapoint_set.all():
+        qs = experiment.datapoint_set.all()
+        if queryset is not None:
+            qs = queryset
```
Comment (tymees) [inline experiments/utils/download.py:147]: This feels weird?

I'd rather set the ``queryset`` variable if it's ``None``:

```python
if queryset is None:
    queryset = experiment.datapoint_set.all()
```

File: experiments/views.py [hunk 3]
```diff
@@ -319,3 +319,20 @@ def get(self, request, experiment, file_format='csv', data_point=None):
...
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['experiment'] = self.experiment
+        return kwargs
+
+    def form_valid(self, form):
+        file_format = form.cleaned_data['file_format']
+        queryset = self.experiment.datapoint_set\
+            .filter(session__experiment_state__in=form.cleaned_data['include_status'])\
```
Comment (tymees) [inline experiments/views.py:336]: More style related, for ease of viewing I'd say a space before the slash is preferable

File: experiments/models.py [hunk 4]
```diff
@@ -183,6 +185,16 @@ def __str__(self):
     session = models.ForeignKey(
         'ParticipantSession', on_delete=models.CASCADE, null=True)
 
+    @property
+    def status_label(self):
```
Comment (tymees) [inline experiments/models.py:189]: I'd say that 'status_label' isn't really convention. Often in Django thos would either be called `status` or `get_status_display()`

File: experiments/locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -320,14 +362,19 @@ msgstr "Experiment created!"
...
+#: experiments/views.py:258
 msgid "experiments:message:delete_all_data:success"
-msgstr "All data deleted!"
\ No newline at end of file
+msgstr "All data deleted!"
+
+#~ msgid "experiments:models:targetgroup:completed:help"
+#~ msgstr ""
+#~ "How many participants from that target group had completed the experiment "
+#~ "(so far)"
```
Comment (tymees) [inline experiments/locale/en/LC_MESSAGES/django.po:380]: This may be cleaned up

File: experiments/utils/download.py (hunk 2 above)
Comment (bbonf) [reply experiments/utils/download.py:147]: That was my first attempt, but the python type checker I use didn't like that I was iterating over an Optional variable. So I had to introduce a second variable whose type is not Optional, and that's why it should have a reasonable default.

**Review by tymees — APPROVED** [review]

**Pull request merged**